runtime/libffi/z/ffi.c

```diff
@@ -114,27 +114,27 @@ ffi_check_struct_type (ffi_type *arg)
   {
     case 1:
     case 2:
-      return FFI_TYPE_UINT32; 
+      return FFI_TYPE_UINT32;
     case 4:
       if (arg->type == FFI_TYPE_FLOAT)
        	return FFI_TYPE_FLOAT;
       else
-       	return FFI_TYPE_UINT32; 
+       	return FFI_TYPE_UINT32;
```

Comment:
Hmm, it complains about this and line 127
```
09:51:27  runtime/libffi/z/ffi.c:122: space before tab in indent.
09:51:27  +       	return FFI_TYPE_UINT32;
09:51:27  runtime/libffi/z/ffi.c:127: space before tab in indent.
09:51:27  +       	return FFI_TYPE_UINT64;
```

I am not sure what change I can make here - I simply removed whitespace at the end of file.